src/utils/frame.js

```diff
@@ -123,6 +124,9 @@ export function getLibraryFromUrl(frame: Frame) {
 }
 
 const displayNameMap = {
+  Babel: {
+    tryCatch: "Async"
```

Comment:
So... This is definetly the best way to do it given that we just match on the first frame's display name. 

I think this is fine to ship, but we might need another way in the future to check all the frames display names and decide based on another name like asyntToGenerator

Reply:
Agreed.